### PR TITLE
[ADD] Module which allow to deactivate partner bank accounts

### PR DESCRIPTION
**Pull request by benwillig**

### CODE REVIEW

**Review by adrienpeiffer — APPROVED** [review]
(Code review)

File: res_partner_bank_active/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+partner banks in sale configuration menu.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
+
+Known issues / Roadmap
+======================
+
+. If you deactivate a partner bank account, linked journals will still be active.
```
Comment (pedrobaeza) [inline res_partner_bank_active/README.rst:41]: s/partner/company

File: res_partner_bank_active/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (pedrobaeza) [inline res_partner_bank_active/README.rst:47]: fill repo

File: res_partner_bank_active/views/res_partner_bank.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<odoo>
+
+    <record model="ir.ui.view" id="res_partner_bank_form_view">
+        <field name="name">res.partner.bank.form (in res_partner_bank_active)</field>
+        <field name="model">res.partner.bank</field>
+        <field name="inherit_id" ref="base.view_partner_bank_form"/>
+        <field name="arch" type="xml">
+
+            <field name="company_id" position="after">
+                <field name="active"/>
```
Comment (pedrobaeza) [inline res_partner_bank_active/views/res_partner_bank.xml:14]: isn't better to follow new v10 approach of an smart-button?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Shouldn't this be called without `res_` prefix for following the rest of the modules `partner_*`? Is this also the best place for this? partner-contact can fit

File: res_partner_bank_active/views/res_partner_bank.xml (hunk 3 above)
Comment (benwillig) [reply res_partner_bank_active/views/res_partner_bank.xml:14]: There is no "button box" divs in the base view, and I don't think this module should add it.

**Comment by benwillig** [comment]
You are right, we can close this PR and I will reopen it in partner-contact repo.

**Comment by benwillig** [comment]
This PR can be closed. Moved to https://github.com/OCA/partner-contact/pull/489

**Pull request closed without merging**